python-regression/tests/features/steps/api_test_steps.py

```diff
@@ -228,3 +233,60 @@ def check_node_sync(step, node, milestone):
         "Latest Solid Milestone {} on {} is not the expected {}".format(latestSolidMilestone, node, milestone)
 
 
+@step(r'A transaction is issued on "([^"]+)" with:')
```

Comment:
This should already be a step in the `transaction_steps.py`, and instead of storing the value in world the way you have here, if you use the existing logic, the results will be stored in the world variable under the `attachToTangle` key as seen here: 
https://github.com/iotaledger/iri/blob/dev/python-regression/tests/features/steps/transaction_steps.py#L14 
see line 31

Reply:
To simplify the `transaction_hash` addition to world, perhaps consider adding something along the lines of: https://github.com/iotaledger/iri/blob/dev/python-regression/tests/features/steps/transaction_steps.py#L89-L91 
to the https://github.com/iotaledger/iri/blob/dev/python-regression/tests/features/steps/transaction_steps.py#L14 call